dfastmi/batch/ConfigurationCheckerLegacy.py

```diff
@@ -73,251 +71,15 @@ def check_configuration(self, rivers: RiversObject, config: configparser.ConfigP
         except ValueError:
             return False
 
-        nwidth = reach.normal_width
-        [_, apply_q, _, _, _, _, _, _] = self.get_levels(reach, config, nwidth)
+        initialized_config = ConfigurationInitializerLegacy(reach, config)
 
         mode_str = config.get("General", "Mode", fallback=DFLOWFM_MAP)
         ret_val  = True
         for i in range(3):
-            if apply_q[i] and not self._validator.is_valid(mode_str, config, i) and ret_val:
+            if initialized_config.apply_q[i] and not self._validator.is_valid(mode_str, config, i) and ret_val:
```

Comment:
Not sure if it is part of this issue or should be tackled in the previous issue..
does the is_valid method anything with logging or messaging the user?
else isnt the ret_val redundant?
e.g.

```
        for i in range(3):
            if initialized_config.apply_q[i] and not self._validator.is_valid(mode_str, config, i):
                return false
        return true
```

Reply:
Yes it is logging per condition. not failing fast, but all conditions are checked and logged if needed